src/node.h

```diff
@@ -17,11 +17,11 @@ typedef struct {
 } cmark_literal;
 
 typedef struct {
-  cmark_list_type list_type;
   int marker_offset;
   int padding;
   int start;
-  cmark_delim_type delimiter;
+  unsigned char list_type;
```

Comment:
I'm just curious (as a C ignoramus) why this is needed.  Does the compiler default to using more than one byte for a cmark_delim_type?

Reply:
The C standard mandates a single implementation-defined type for `enum`s. Compilers typically use `int`.

Reply:
oh, that's good to know.